packages/listbox/src/ListboxMixin.js

```diff
@@ -252,6 +252,7 @@ const ListboxMixinImplementation = superclass =>
       super.firstUpdated(changedProperties);
 
       this.__moveOptionsToListboxNode();
+      this._initialCheckedIndex = this.checkedIndex;
```

Comment:
Why not save modelValue instead (we do it in other places as wel, since this is SSOT and all other values are derived from it)?

Reply:
Since we loop through all child form elements to reset the old checked state, which goes the easiest with `index`

Reply:
But using the `modelValue` makes all other code way easier. So i'll update it